### ISSUE TITLE
问题链接合集

### ISSUE THREAD

**Issue by NarcissusInMirror** [NarcissusInMirror/DailyWorkLog#2]
#### 问题链接合集

* [Python什么情况下会生成pyc文件](https://www.zhihu.com/question/30296617/answer/112564303)

* [Python的运行机制](https://blog.csdn.net/jeff_liu_sky_/article/details/52097060)

* [谈谈 Python 程序的运行原理](https://www.restran.net/2015/10/22/how-python-code-run/) Mac OS X 下使用 Ruby Gem 的两个坑 这篇文章的意义不仅在于他解决的问题 中间有一段关于系统自带和自己安装的软件的差别 十分有助于对于系统的理解！

* [Git 一些错误的解决方法(git add -u)](http://blog.51cto.com/halolk/1304701)

* [如何快速搭建自己的github.io博客](https://keysaim.github.io/post/blog/2017-08-15-how-to-setup-your-github-io-blog/)

* [Anaconda 安装OpenCV](https://blog.csdn.net/castle_cc/article/details/78809931)

* [全面的thresholding，值得一试](https://docs.opencv.org/3.4/d7/d4d/tutorial_py_thresholding.html)

* [Morphological Transformation](https://docs.opencv.org/master/d9/d61/tutorial_py_morphological_ops.html#gsc.tab=0)

* [Mac上传文件到Linux服务器](https://www.jianshu.com/p/1afd25e7459d)

* [SCP](http://www.runoob.com/linux/linux-comm-scp.html)
`scp -r Desktop/filtered/ qiushuhao@10.106.128.95:/home/qiushuhao`
`scp -r qiushuhao@10.106.128.95:/home/qiushuhao Desktop/filtered/ `

* conda install —file FILEPATH 按照文件安装相应pacakage

* [Linux查看GPU信息和使用情况](https://blog.csdn.net/dcrmg/article/details/78146797)

* [卷积神经网络——输入层、卷积层、激活函数、池化层、全连接层](https://blog.csdn.net/yjl9122/article/details/70198357)

* [深度学习框架比较（Caffe, TensorFlow, MXNet, Torch, Theano）](https://blog.csdn.net/ZYC88888/article/details/79166190)

* [Python Virtual Environments: A Primer](https://realpython.com/python-virtual-environments-a-primer/)

* [TensorFlow strides 参数讨论](https://blog.csdn.net/lanchunhui/article/details/61615714)

* [TensorFlow中CNN的两种padding方式“SAME”和“VALID”](https://blog.csdn.net/wuzqChom/article/details/74785643)

* [tensorflow的reshape操作tf.reshape()](https://blog.csdn.net/m0_37592397/article/details/78695318)

* [Epoch vs Batch Size vs Iterations 1](https://towardsdatascience.com/epoch-vs-iterations-vs-batch-size-4dfb9c7ce9c9)
[Epoch vs Batch Size vs Iterations 2](https://www.zhihu.com/question/32673260)

* [图片数据集太少？看我七十二变，Keras Image Data Augmentation 各参数详解](https://zhuanlan.zhihu.com/p/30197320)

* [Python 优雅的操作字典](https://www.linuxzen.com/python-you-ya-de-cao-zuo-zi-dian.html)

* [Batch Normalization 学习笔记](https://blog.csdn.net/hjimce/article/details/50866313)

* [Step by Step, A Tutorial on How to Feed Your Own Image Data to Tensorflow](https://yeephycho.github.io/2016/08/15/image-data-in-tensorflow/)

* [Hello, TensorFlow!](https://www.oreilly.com/learning/hello-tensorflow)

* [Python字符串格式化（%s，%d都是什么）](http://www.cnblogs.com/vamei/archive/2013/03/12/2954938.html)

* [*args 和 **kwargs](https://eastlakeside.gitbooks.io/interpy-zh/content/args_kwargs/)

* [TensorFlow指定特定GPU以及占用显存的比例](https://blog.csdn.net/m0_37041325/article/details/77488981)
`os.environ["CUDA_VISIBLE_DEVICES"]=‘6‘’，‘7’`

* [机器学习数据预处理——标准化/归一化方法](https://www.cnblogs.com/bjwu/p/8977141.html)

* [What exactly are iterator, iterable, and iteration?](https://stackoverflow.com/questions/9884132/what-exactly-are-iterator-iterable-and-iteration)

* [常见向量范数和矩阵范数](https://blog.csdn.net/left_la/article/details/9159949)

* [深度学习VGG模型核心拆解](https://blog.csdn.net/qq_40027052/article/details/79015827)

* [矩阵补全（matrix completion）的经典算法有哪些？目前比较流行的算法是什么？](https://www.zhihu.com/question/47716840)

* [机器学习中的范数规则化之（二）核范数与规则项参数选择(rank(W)近似为核范数、矩阵填充、robust pca）](https://blog.csdn.net/zouxy09/article/details/24972869)

* [详解 MNIST 数据集](https://blog.csdn.net/simple_the_best/article/details/75267863)

* [Tensorflow加载预训练模型和保存模型1](https://blog.csdn.net/huachao1001/article/details/78501928)
[Tensorflow加载预训练模型和保存模型2](https://www.tensorflow.org/guide/saved_model?hl=zh-cn)
[Tensorflow加载预训练模型和保存模型3](https://www.tensorflow.org/guide/saved_model)

* [Tensorflow的可视化工具Tensorboard的初步使用](https://blog.csdn.net/sinat_33761963/article/details/62433234)
`$ tensorboard —logdir=‘/Users/qiushuhao/Desktop’`

* [keras解决多标签分类问题](https://zhuanlan.zhihu.com/p/34712246)

* [Linux中查看磁盘空间和文件大小的方法](https://blog.csdn.net/lkforce/article/details/52859572)

* [linux&windows utf8和gbk编码识别&互相转换1](https://blog.csdn.net/u010821666/article/details/78611504)
[linux&windows utf8和gbk编码识别&互相转换2](https://codingstandards.iteye.com/blog/807077)

* [Linux查看进程](https://linuxtools-rst.readthedocs.io/zh_CN/latest/tool/ps.html)
`ps -ef|grep PID`

* [numpy数据类型](https://docs.scipy.org/doc/numpy-1.13.0/user/basics.types.html)

* [Markdown中插入数学公式的方法1](https://blog.csdn.net/xiahouzuoxin/article/details/26478179)
[Markdown中插入数学公式的方法2](https://juejin.im/post/5a6721bd518825733201c4a2)
[Markdown中插入数学公式的方法3](https://jzqt.github.io/2015/06/30/Markdown%E4%B8%AD%E5%86%99%E6%95%B0%E5%AD%A6%E5%85%AC%E5%BC%8F/)

* 本地预览博客 `$ jekyll serve --watch`

* [计算机视觉会议](https://blog.csdn.net/chlele0105/article/details/17204163)

* [linux上anaconda的卸载](https://blog.csdn.net/qq_22474567/article/details/54984257)
`最后一步重新打开Terminal很重要！！！！`

* [.bash_profile和.bashrc的什么区别及启动过程](https://www.cnblogs.com/liduanjun/p/3536993.html)

* [Linux的五个查找命令](http://www.ruanyifeng.com/blog/2009/10/5_ways_to_search_for_files_using_the_terminal.html)

* [Mac设置快捷键](https://blog.csdn.net/myth13141314/article/details/74171006)

* [keras入门 --- Data augmentation(数据扩充)](https://blog.csdn.net/hnu2012/article/details/54017564)

* [python文件创建,删除,移动,复制,重命名](https://blog.csdn.net/u011961856/article/details/79122057)

* [TensorFlow 之基于Inception V3的多标签分类 retrain](https://blog.csdn.net/u012936765/article/details/76944727)

* `将unix executable文件放到 /usr/local/Cellar下 在bash中可以找到`

* [关于TCGA（癌症基因组数据集）使用指南](https://www.jianshu.com/p/829c3e311e54)

￼* [Mac桌面不显示东西](https://blog.csdn.net/Mr_OOO/article/details/71335483)

* [GPU，CUDA，cuDNN的理解](https://blog.csdn.net/u014380165/article/details/77340765)

* [cuda、cudnn、驱动版本查看及nvidia驱动、cuda安装及卸载](https://blog.csdn.net/u013187057/article/details/81475806)

* `pip install SomePackage==1.0.0` 修改某个包的版本

￼* https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html

* [Python读写文本文件](http://funhacks.net/explore-python/File-Directory/text_file_io.html)

* [Keras添加AUC metrics（继承callback类）1](https://blog.csdn.net/qq_22885109/article/details/81081977)
[Keras添加AUC metrics（继承callback类）2](https://keras.io/zh/metrics/)

* [Keras加载模型](https://keras.io/zh/models/about-keras-models/ )

* [wget 使用指南1](http://math.ecnu.edu.cn/~jypan/Teaching/Linux/command/wget.htm)
[wget 使用指南2](https://blog.csdn.net/RichardYSteven/article/details/4565931)

* [ASCII码、Unicode（统编）、UTF-8、GB国标码的比较](https://blog.csdn.net/farrellcn/article/details/6593502)

* [Multi-label image classification with Inception net](https://towardsdatascience.com/multi-label-image-classification-with-inception-net-cbb2ee538e30)

* [An Overview of ResNet and its Variants](https://towardsdatascience.com/an-overview-of-resnet-and-its-variants-5281e2f56035)

* [Avoiding headaches with tf.metrics](http://ronny.rest/blog/post_2017_09_11_tf_metrics/)

* [神经网络不收敛](http://theorangeduck.com/page/neural-network-not-working)

* [Linux 技巧：让进程在后台可靠运行的几种方法](https://www.ibm.com/developerworks/cn/linux/l-cn-nohup/index.html)
```
$ setsid python a.py > a.txt &
此条命令创建父进程为1的进程，不会因为ssh中断而中断，并将输出重定向至a.txt中，&将命令放在后台运行，并返回一个进程ID。值得注意的是，此条命令下返回的id用ps -ef|grep 无法找到，需将ID+1才是真正的进程ID 原因未知.
```
* [服务器上PS命令后各列含义](https://blog.51cto.com/20100125/587097)
`C:处理器使用率百分比`
